src/services/role/utils/paginate.js

```diff
@@ -0,0 +1,14 @@
+const paginate = (result = [], { $limit, $skip = 0 } = {}) => {
+	if (!Array.isArray(result)) {
+		return result;
+	}
+	const data = result.slice($skip, $limit ? $skip + $limit : $limit);
+	return {
+		total: data.length,
+		limit: $limit || result.length || data.length,
+		skip: $skip,
+		data,
+	};
+};
+
+module.exports = paginate;
```

Comment:
Duplikat von `src/utils/array.js`

Reply:
Schaue ich mir an, ich wusste nicht das ein fertiger Stand in utiles liegt. Können wir das aus dem logischen Umfeld Arrays raus ziehen und paginate.js ablegen.

Reply:
klar

Reply:
Das klinkt jetzt komisch, aber wenn ich austausche failed er und ich finde nicht raus warum. 
Ich werde das nach dem der Haupt PR durch ist noch mal untersuchen. Die in / out puts und das handling sieht identisch aus. o_O